tests/unit/dstoreMemory/Test.js

```diff
@@ -66,7 +61,11 @@ define([
 		},
 		"test initial view": function () {
 			this.timeout = 20000;
-			return new Application(JSON.parse(stripComments(dstoreMemoryconfig1)), dstoreMemoryContainer1)
+			var d = this.async(10000);
+			if (has("ie") === 10) {
+				this.skip("Skipping dstoreMemory tests on IE10");
```

Comment:
I was actually hoping you'd list why you were skipping the tests.   Like a ticket number or something.